spec.html

```diff
@@ -24033,8 +24033,7 @@ <h1>Object.assign ( _target_, ..._sources_ )</h1>
         <p>The `assign` function is used to copy the values of all of the enumerable own properties from one or more source objects to a _target_ object. When the `assign` function is called, the following steps are taken:</p>
         <emu-alg>
           1. Let _to_ be ? ToObject(_target_).
-          1. If only one argument was passed, return _to_.
-          1. Let _sources_ be the List of argument values starting with the second argument.
+          1. If _sources_ is empty, return _to_.
```

Comment:
is `sources` a list, or an array (as it would be in code)? if the latter, is "is empty" defined for arrays?